modules/ossm-config-operator-infrastructure-node.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * service_mesh/v2x/ossm-deployment-models.adoc
+
+:_content-type: PROCEDURE
+[id="ossm-config-operator-infrastructure-node_{context}"]
+== Configuring the OpenShift Service Mesh Operator to run on infrastructure nodes
+
+This task should only be performed if the {SMProductShortName} Operator will run on an infrastructure node. If the operator will run on a worker node, skip this task.
+
+.Prerequisites
+
+* The Service Mesh Operator must be installed using the Operator hub.
+
+* You must mark one of the worker nodes as an infrastructure node. For more information, see Creating infrastructure machine sets.
```

Comment:
Not sure whether we consider infra nodes a sub-type of worker nodes or a separate type of node. In any case, it might make sense to change the sentence to read "... one of the nodes as an infrastructure node".

Reply:
Revised the bullet to say "One of the nodes must be an infrastructure node."